### ISSUE TITLE
Script error trouble

### ISSUE THREAD

**Issue by AJOgames** [icefallgames/SCICompanion#14]
for some reason, when I try to open a game, (nut running the game, but opening the game in SCI Companion) I get a message saying "error enumerating items corrupted resource header-mismatched types, so I double clicked on the bottom left corner of the screens to see what exactly went wrong, (via the little yellow pop-up) and then a little window appeared, and the screenshot below should show you what it shows,

![capture](https://user-images.githubusercontent.com/42022618/43622449-09d37ee2-9691-11e8-9ee0-da836724f9ae.PNG)
(albeit the whole "1 script compiled" message wasn't there when I first tried to open it, the reason why it's there in the screenshot  is because I was trying to figure out what to do to fix it,)

I don't know what to do, all I know is that it has to do with a new script I created yesterday called "title.sc"

Now, I don't know if you the main one handling the whole SCICompanion  thing, on the website there doesn't seem to be a contact email, closest thing to that is the social media links

**Comment by icefallgames** [comment]
It probably means your resource.001 (or resource.000? I forget) and resource.map files are out of sync. resource.map contains the list of resources, and then the actual resource data is in resource.001. This could happen if SCICompanion can't delete one of the older ones (say if an antivirus has than open or something). Do you see any resource.001.bak or resource.map.bak files in that folder?

**Comment by icefallgames** [comment]
Also, probably the best place for support are the forums here: http://sciprogramming.com/, I don't actively develop SCICompanion anymore.

**Comment by AJOgames** [comment]
yes I do see a resource.001.bak file, but no resource.map.bak file, just a
regular resource.map file,

Going off-topic here, before I ran into SCI Studio and  SCICompanion, I
first tried developing games using the AGI Engine, using an old (and quite
buggy) WinAGI, but that thing was very picky about what you could do with
it, so I just decided to quit using it, seeing as SCI is superior anyways

It also appears as though you can't import wav files into SCICompanion, at
least if you're using the SCI0 template, although

**Comment by AJOgames** [comment]
I found the "game.ini" file, and opened it in Notepad, and I noticed it has
a lot of the missing views/pics I had created, is there any way I can get
them back into the game again?

**Comment by AJOgames** [comment]
I figured it out! apparently the resource.map somehow turn into "resource.map.old" I don't know why or how, but all I did was simply take out the ".old" and it worked, now I can continue making my game!

Does SCICompaion support CGA? I know some SCI games (like the original Hero's Quest) support CGA, as well as tandy graphics and Hercules graphics, just wandering.

**Comment by icefallgames** [comment]
Glad you figured it out. SCICompanion doesn't do anything to specifically support CGA, but it doesn't do anything to prevent it either. I think it's a driver thing. Anyway, your questions are probably best asked on sciprogramming.com. In fact:
http://sciprogramming.com/community/index.php?topic=1673.0

**Comment by AJOgames** [comment]
Sorry to keep bothering you like this, but sciprogramming.com right now is
a bit empty, not much activity going on over there,
What script, based on the SCI.0 template, handles the main Ego/player
movement? (like Up, Down Left, Right, Spacebar, etc)
these are the specs, SCI.0, SCI Studio(script language)
I'm assuming its MAIN,sc, but I can't seem to find where exactly the code
for the ego code is...
The reason I ask is that I wanted to add some new gameplay mechanics, (like
press CTRL to attack, press ALT to jump, press Shift to run, that sort of
thing)

**Comment by icefallgames** [comment]
sciprogramming.com is fairly active, you should get a response pretty quick there.
